css/popup.css

```diff
@@ -0,0 +1,162 @@
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+a {
+  color: #FFF;
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0px;
+  padding-left: 0px;
+}
+
+ul li span {
+  float: right;
+  margin-top: 9px;
+  font-size: 13px !important;
+  border: 2px solid rgba(255, 255, 255, 0.7);
+  display: inline-block;
+  text-align: center;
+  height: 30px;
+  font-weight: bold;
+  width: 30px;
+  line-height: 25px;
+  border-radius: 15px;
+}
+.close {
+  outline: none;
+  background-color: transparent;
+  border: 0px;
+  width: 20px;
+  height: 20px;
+  position: absolute;
+  top: 12px;
+  right: 15px;
+  cursor: pointer;
+  opacity: 0;
+  transition: all 0.4s cubic-bezier(0.4, 0, 0.2, 1) 0s;
+  transform: translateX(-60px) rotate(90deg);
+}
+.close.active {
+  transition: all 0.4s cubic-bezier(0.4, 0, 0.2, 1) 0.2s;
+  opacity: 1;
+  transform: translateX(0px) rotate(360deg);
```

Comment:
It does not have an z-index of 2 or something so it is under the content, i can not see it when i expand the modal window